src/accordion.tsx

```diff
@@ -194,19 +199,22 @@ const AccordionItemContent = React.forwardRef<TouchableElement, AccordionItemCon
 );
 
 export const AccordionItem = React.forwardRef<TouchableElement, AccordionItemContentProps>(
-    ({dataAttributes, ...props}, ref) => (
-        <AccordionItemContent
-            {...props}
-            ref={ref}
-            dataAttributes={{'component-name': 'AccordionItem', ...dataAttributes}}
-        />
+    ({dataAttributes, role, ...props}, ref) => (
+        <div role={role} className={sprinkles({width: '100%'})}>
```

Comment:
instead of adding a new div just for the role, why don't you forward the `role` prop to `AccordionItemContent`?

Reply:
Discussed offline. In non-boxed rows, we need to set the listitem to the wrapper. Otherwise, we would have a11y problems